TYPO3.Flow/Classes/TYPO3/Flow/Object/DependencyInjection/ProxyClassBuilder.php

```diff
@@ -243,35 +243,43 @@ protected function buildConstructorInjectionCode(Configuration $objectConfigurat
     {
         $assignments = array();
 
-        $argumentConfigurations = $objectConfiguration->getArguments();
+        $configurationArguments = $objectConfiguration->getArguments();
```

Comment:
Agree, BUT the object is called `ConfigurationArgument` I just wanted to harmonize that...